MCMC/MCMC-flow/project.py

```diff
@@ -131,6 +134,33 @@ def sample(job):
         print("Simulation finished completed")
         print("-----------------------------")
 
+@MyProject.operation
+@MyProject.pre(sampled)
+@MyProject.post(analyzed)
+def analysis(job):
+    from utils import avg_nn, rdf
+    import numpy as np
+    os.makedirs(os.path.join(job.ws, "analysis/rdf"))
+    gsdfile = job.fn('trajectory_1.gsd')
+    rdf = rdf(gsdfile, start=-30)
```

Comment:
Please apply this change to the rdf function in utils.py

`rdf.compute((box,points), reset=False)`

The default value for reset is True, which means if you are computing rdf from multiple snapshots, freud doesn't accumulate the computated values from different snapshots. When reset is True, the result rdf is only from last snapshot.